Add `Intermediate Global Taiko Showdown 2021` wiki article (#7092)
ppy/osu-wiki: wiki/Tournaments/GTS/IGTS_2021/en.md

4. Failed scores will count.
5. Only two players from each team will be playing per map. There are no limitations on player swapping.
6. Seeding will be done with the average rank of each team on each map.
7. Maps will be played twice in a row during the lobby. The best run will be taken into account for the ranking.
8. Only the 32 best teams will continue on to the bracket stage.

### Mappool instructions

1. There will be one mappool for each stage.
2. The loser's bracket will play on the same pool as the winner's bracket of the same weekend.
3. The Qualifier pool will be different from all other rounds, as it will have a format of 2 NoMod maps, 1 Hidden map, 1 HardRock map, 1 DoubleTime map, and 1 FreeMod map.
4. Each mappool consists of 6 sections: **NoMod, Hidden, HardRock, DoubleTime, FreeMod, and Tiebreaker**.
5. Total maps per stage will be listed as follows:
   - The Group stage mappool will consist of 14 maps.
   - The Round of 16 and Quarterfinals mappools will consist of 15 maps.
   - The Semifinals, Finals and Grand Finals mappools will consist of 16 maps.
6. All stages will have 2 Hidden, HardRock, and DoubleTime maps, and 3 FreeMod maps.

> 7. The FreeMod pool is played with FreeMod activated. Every player **has to pick** Hidden, HardRock, or both. At least one player of each team must have at least one mod activated.

Reviewer: > does this mean that you *can't* use no mods (second sentence) or that you *can* use no mods (third sentence)?

mind checking this? if it's the former, then the third sentence should be removed, and if it's the latter, then it would look like this

```suggestion
7. The FreeMod pool is played with FreeMod activated. Every player may pick a combination of Hidden and HardRock, but at least one player of each team must have at least one mod activated.
```

Author: Should be latter, they can use No Mods.

3. We will try to schedule as good as possible to fit with the timezones. The schedule of each stage will be released on the weekend prior to it.
4. Reschedules will only be considered if both teams agree to a time, this needs to be done and notified to the tournament staff before **Thursday at 23:59 UTC** in that particular week when the match takes place.
5. Reschedules may only be requested by **the team captain**.

### Match regulations

1. A private lobby will be created 10 minutes before the match started by a referee. Both team's captains will be invited into the room when it's match time. Then the referee or the captains will invite the other players into the room.
2. If no staff or referee is available the match will be postponed.
3. If less than the minimum required players attend, the maximum time the match can be postponed is **10 minutes**.
4. There is no penalty for not joining the room instantly after match time as long as it's within the postponed time.
5. If neither of both teams have the minimum required players after 10 minutes it will count as a loss for both teams in Group Stage. In the double elimination bracket the highest seeded team will advance.

> 6. The match will be played as TeamVs and ScoreV2 and the name of the room must be: "IGTS 2021: (Team Red) vs (Team Blue)".

Reviewer: ```suggestion
6. The match will be played with Team VS and Score v2, and the name of the room must be: `IGTS 2021: ({Team Red}) vs ({Team Blue})`.
```

7. Each team is allowed one warm-up but it must be **below 3 minutes**, and must be sent to the referee and streamer of the match one hour prior to the match's start time. Otherwise, the team will have no warmups.

> 8. If a team does not respect the time said in the rule above, they will not have any warm-up for their match.

Reviewer: ```suggestion
8. Submitting a warmup less than one hour prior to the match will result in no warmup.
```

> 9. Each captain has to ban **one beatmap** from the pool (apart from Tiebreaker). These beatmaps are not allowed to be picked by any team in the entire match.

Reviewer: ```suggestion
9. Each captain has to ban **one beatmap** from the pool (apart from tiebreaker). These beatmaps are not allowed to be picked by any team in the entire match.
```

> 10. The captain of each team will roll with the command `!roll` in the chat.

Reviewer: ```suggestion
10. The captain of each team will roll with the `!roll` command in the chat.
```

11. The team with the highest roll **bans second and picks first**. The team with the lowest roll **bans first and picks second**.

> 12. Each captain has 2 mins to announce their ban. If they didn't tell the referee by that time the ban will be made with a roll.

Reviewer: ```suggestion
12. Each captain has 2 minutes to announce their ban. If they do not tell the referee by that time, the ban will be made with a `!roll`.
```

13. Beatmap selection will then alternate between each captain selecting a beatmap out of the map pool.
14. Captains may pick freely from any bracket.

> 15. Teams will have 2 minutes to pick a beatmap and 2 minutes to get ready. If a team didn't tell their pick before the end of the timer the map will be picked with a roll.

Reviewer: ```suggestion
15. Teams will have 2 minutes to pick a beatmap and 2 minutes to get ready. If a team does not pick before the end of the timer, the map will be picked with a `!roll`.
```

16. Exchanging players during a match is allowed without limitations.

> 17. If a player disconnects between the beatmaps and the team cannot provide an exchange the match can be delayed 10 minutes at most. This delay can only be applied once.

Reviewer: ```suggestion
17. If a player disconnects between beatmaps and the team cannot provide a substitute, the match can be delayed 10 minutes at most. This delay can only be applied once.
```

> 18. If a player disconnects, they get treated as if they failed the map.

Reviewer: ```suggestion
18. Disconnections are treated as failing the map.
```

> 19. Disconnects within 30 seconds after the beatmap was started allows the map to be replayed. This is only allowed once per team.

Reviewer: ```suggestion
19. A beatmap can be replayed upon a disconnection that happened within 30 seconds of starting. This is only allowed once per team.
```

20. Failed players' scores do get added to the team score.

> 21. If the beatmap ends in a draw the game will be nullified and the map will be replayed.

Reviewer: ```suggestion
21. If the beatmap ends in a draw, the game will be nullified and the map will be replayed.
```

22. In case of a tie in points (example: 4-4 in BO9) the tiebreaker is selected.

### All Stars rules

> 1. All Stars is a fun event taking place on the same weekend as the Grand Finals! It'll showcase the best players of the tournament playing against each other in a special match.

Reviewer: ```suggestion
1. All Stars is a fun event taking place on the same weekend as the Grand Finals! It will showcase the best players of the tournament playing against each other in a special match.
```

2. 12 players will participate in this event. They will be decided by a vote.

> 3. The top 12 players will play in All Stars. If there are more than 2 players from a country in the top 12 most voted players only the 2 best will partake in All Stars.

Reviewer: ```suggestion
3. If there are more than 2 players from a country in the top 12 most voted players, only the 2 best will partake in All Stars.
```

4. The top 2 players will be captains of each team. Their teammates will be decided by the ranking of the next 6 players. The topmost voted player will play with players with an odd number and the second most voted player with players with an even number.
5. The mappool will be made by a special team of mappoolers.
6. The event will be a 4 vs 4 match with the following rules:
   - Best-of-15 format (First to 8 points).

>    - All the maps are under true FreeMod rule meaning players are free to pick any of the following mods or a combination of them: NoMod, Hidden, HardRock.

Reviewer: ```suggestion
   - All the maps are played under FreeMod rules, meaning players are free to pick any of the following mods or a combination of them: NoMod, Hidden, HardRock.
```

   - No bans.
   - Map selection is the same as a normal match.

>    - Same rules as a normal match for disconnections.

Reviewer: ```suggestion
   - The same rules as normal matches apply regarding disconnections.
```
